### PR TITLE
Extract shared methods to expose common structure

### PR DESCRIPTION
**Pull request by c-w**
Not sure why we had three different ways of getting the mainEdge before. Would this refactor make sense?

### CODE REVIEW

**Review by Smarker — APPROVED** [review]
LGTM

File: src/components/Insights/Dashboard.js [hunk 1]
```diff
@@ -145,7 +158,7 @@ export const Dashboard = React.createClass({
       <div key={'locations'} className="doughnutChart">
         <GraphCard cardHeader={cardHeader}>
           <PopularLocationsChart
-            mainEdge={this.state.categoryValue["name_"+this.state.language]}
+            mainEdge={this.getMainEdge()}
```
Comment (erikschlegel) [inline src/components/Insights/Dashboard.js:161]: I think you're going to run into translation issues here?

**Review by erikschlegel — APPROVED** [review]
LGTM

File: src/components/Insights/Dashboard.js (hunk 1 above)
Comment (c-w) [reply src/components/Insights/Dashboard.js:161]: The method this delegates to already takes care of localization :)

```js
getMainEdge() {
  if (!this.state.categoryValue) {
    return undefined;
  }

  return this.state.categoryValue[`name_${this.state.language}`] || this.state.categoryValue.name;
}
```

**Pull request merged**